docs/guides/1238-making-reusable-package.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /1238/making-reusable-package
+displayed_sidebar: "0.2"
+---
+
+# Making reusable packages
+
+Whilst splitting your plan into several files is a good idea, you will sometimes need to create standalone packages aiming to be reusable and shared. Let's explore how to do that.
+
+## Dagger project vs packages
+
+Understanding the difference between a project and a package is an important distinction, as it will help organizing your work.
+
+### Dagger project
+
+- Are use-case oriented: they solve a specific need through a set of actions
+- Aims to host a Dagger Plan
+- Contains a `cue.mod` folder
+
+### Package
+
+- Are reusable actions that may be useful for several use-cases
+- Aims to be shared across several projects
```

Comment:
Sharing accross several projects requires a module. The module concept should be clear.